packages/jsapi-utils/src/ViewportDataUtils.ts

```diff
@@ -74,11 +74,11 @@ export function createOnTableUpdatedHandler<T>(
  */
 export function defaultRowDeserializer<T>(
   row: ViewportRow,
-  columns: Column[]
+  columns: dh.Column[]
 ): T {
   return columns.reduce((result, col) => {
     // eslint-disable-next-line no-param-reassign
-    result[col.name as keyof T] = row.get(col);
+    result[col.name as keyof T] = row.get(col) as T[keyof T];
```

Comment:
We do have the `ValueOf` utility type for this case, e.g.
```
    result[col.name as keyof T] = row.get(col) as ValueOf<T>;
```
Doesn't really matter.